src/test/java/com/commercetools/dataimport/ImportJobIntegrationTest.java

```diff
@@ -11,13 +11,10 @@
     @Test
     public void runs() {
         assertThat(fetchTotalCategories()).as("Categories are imported").isEqualTo(131);
-        assertThat(fetchTotalProducts()).as("Products are imported").isEqualTo(2);
-        assertThat(fetchTotalChannels()).as("Channels are imported").isEqualTo(18);
         assertThat(fetchTotalTypes("order")).as("Order types are imported").isEqualTo(1);
         assertThat(fetchTotalTypes("customer")).as("Customer types are imported").isEqualTo(1);
         assertThat(fetchTotalTypes("channel")).as("Channel types are imported").isEqualTo(1);
         assertThat(fetchTotalOrders()).as("Orders are imported").isEqualTo(2);
-        assertThat(fetchTotalInventory()).as("Inventory is imported").isEqualTo(4);
```

Comment:
The new flow structure looks good! Shouldn't we now be able to enable these 3 assertions again?

Reply:
Yes, we can! actually already done.

Reply:
Awesome! Then it all looks good to me. I'll accept the changes :)